spec/controllers/projects_controller_spec.rb

```diff
@@ -40,18 +40,6 @@
         expect(assigns(:projects)).to include(project)
       end
 
-      it 'hides volunteer action item' do
-        Project.update_all(accepting_volunteers: true)
-        get :index
-        expect(response.body.scan('sign up to volunteer').size).to be > 0
-      end
-
-      it 'shows volunteer action item' do
-        Project.update_all(accepting_volunteers: false)
-        get :index
-        expect(response.body.scan('sign up to volunteer').size).to eq(0)
-      end
```

Comment:
This is testing for presence/absence sign-up button on the index page - but I don't think we expect users to sign-up on the index page (there isn't much space for a sign up button)
![image](https://user-images.githubusercontent.com/27900457/81061546-71af2e00-8ecc-11ea-9681-842b3498cc08.png)